client/src/App.js

```diff
@@ -1,23 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
 import { BrowserRouter, Route, Switch, withRouter } from "react-router-dom";
 import "./App.css";
 import Nav from "./components/Nav/Nav";
 import Search from "./components/Search/Search";
 import List from "./components/List/List";
+// useContext 
+import {BusinessContext} from "./components/BusinessContext/BusinessContext";
 import About from "./components/About/About"
 import Team from "./components/Team/Team"
 import BizCard from "./components/BizCard/BizCard"
 
 function App() {
+
```

Comment:
you should set state here and pass the values to your `BusinessContext.Provider` below.
```suggestion
  const [businesses, setBusinesses] = useState([]);
```

Reply:
Thx! I added the code.